.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea/
```

Comment:
This should be part of the global config, IMO.

Reply:
Done.